ci/scripts/run-fpga-cw310-tests.sh

```diff
@@ -23,6 +24,9 @@ export BITSTREAM="--offline --list ${SHA}"
 # in case we've crashed the UART handler on the CW310's SAM3U
 trap 'python3 ./util/fpga/cw310_reboot.py' EXIT
 
+readarray -t test_rom_tests < <(./bazelisk.sh query 'rdeps(//...,@bitstreams//:bitstream_test_rom)')
```

Comment:
The two `<` symbols threw me — this is subtle! Could you maybe add an explanatory comment above this line?

I guess `<(./bazelisk.sh query 'rdeps(//...,@bitstreams//:bitstream_test_rom)')` is process substitution, so that part is equivalent to a file descriptor. The first `<` redirects the file to `readarray`'s stdin?

Reply:
I added a comment to explain how we're using input redirection and process substitution and why this is necessary.